Lombiq.Tests.UI/BasicOrchardFeaturesTesting/MediaOperationsTestingUITestContextExtensions.cs

```diff
@@ -32,7 +32,7 @@ public static class MediaOperationsTestingUITestContextExtensions
                 context.Exists(By.XPath($"//span[contains(text(), '{imageName}')]"));
 
                 await context
-                    .Get(By.CssSelector($"a[href=\"/media/{imageName}\"]").OfAnyVisibility())
+                    .Get(By.CssSelector($"a[href=\"{context.UrlPrefix}/media/{imageName}\"]").OfAnyVisibility())
```

Comment:
This breaks if the image has a version query string (for cache invalidation), which will be the default behavior starting with OC 2.0. (it's already like that in [`task/system-text-json-migration`](https://github.com/Lombiq/UI-Testing-Toolbox/blob/task/system-text-json-migration/Lombiq.Tests.UI/BasicOrchardFeaturesTesting/MediaOperationsTestingUITestContextExtensions.cs#L35)) So instead of exact match you should check if the href starts with that path.
```suggestion
                    .Get(By.CssSelector($"a[href^=\"{context.UrlPrefix}/media/{imageName}\"]").OfAnyVisibility())
```